docs/UserGuide.md

```diff
@@ -178,6 +180,7 @@ Example<br>
 
 [Back to Table of Contents](#table-of-contents)
 
+### Add Commands
```

Comment:
This can be aligned more to the header in the table of contents. (Change from `Add Commands` to `Adding a Conntact`

Reply:
Similar for `Delete Commands`, `List Commands`, `Edit Commands`, `View Commands` and `Find Commands`.